client-mvp-04/src/components/createAccount/createAccountContainer.test.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import CreateAccountContainer from './createAccountContainer';
+import { MemoryRouter } from 'react-router-dom';
+import { cleanup, fireEvent, render, screen, wait } from '@testing-library/react';
+import userReducer from '../../store/reducers/userReducer';
```

Comment:
Your component doesn't connect to redux store. This import is redundant. Please, remove it.